rasa/core/actions/action.py

```diff
@@ -147,41 +125,53 @@ def is_retrieval_action(action_name: Text, retrieval_intents: List[Text]) -> boo
     )
 
 
-def action_from_name(
-    name: Text, domain: Domain, action_endpoint: Optional[EndpointConfig]
+def action_for_name_or_text(
+    action_name_or_text: Text, domain: Domain, action_endpoint: Optional[EndpointConfig]
 ) -> "Action":
     """Retrieves an action by its name.
 
     Args:
-        name: The name of the action.
+        action_name_or_text: The name of the action.
         domain: The current model domain.
         action_endpoint: The endpoint to execute custom actions.
 
+    Raises:
+        ActionNotFoundException: If action not in current domain.
     Returns:
         The instantiated action.
     """
+
+    if action_name_or_text not in domain.action_names:
```

Comment:
is it still `domain.action_names`? Can we rename this one as well?

Reply:
We could shorten it to just `actions`, what do you think? I'd probably keep the old one and deprecate it's usage.
We might want to consider doing this change once everything is merged to master to not increasing the diff unnecessarily.

Reply:
`actions` would be confusing as well, since `actions` represent actual actions `Action` class or `ActionExecuted` event

Reply:
at this point I wouldn't worry about the diff, it is skyrocket anyhow 🙈